cpg-console/src/main/kotlin/de/fraunhofer/aisec/cpg/console/CompilationDatabase.kt

```diff
@@ -59,7 +59,6 @@ class CompilationDatabase : Plugin {
                     "import de.fraunhofer.aisec.cpg.graph.allChildren",
                     "import de.fraunhofer.aisec.cpg.graph.ast",
```

Comment:
I think we need to include `import de.fraunhofer.aisec.cpg.graph.*` here I think  and it looks like something is still using `byName`.

Reply:
Its already there:
https://github.com/Fraunhofer-AISEC/cpg/pull/1618/files#diff-2ceb1018c52a5beec418b3ab986e10abe7a5ebda5469c0900af841e9b0ef050aL53

Reply:
Does `byName` still exist? I cant find the symbol anymore.

Reply:
Not sure, but the test fails now :D

Reply:
Ah! you removed one line and the test checks the number of lines. So we need to adjust the assert from 19 to 18.

Reply:
ah, should be fixed now